### DISCUSSION TITLE
FEATURE｜开发文档 或项目Wiki

### DISCUSSION THREAD

**Discussion by StarryCamile** [MCSLTeam/MCSL2#78] [功能建议]
- [x] 未在 [Issues](https://github.com/LxHTT/MCSL2/issues) 中看到其他相同功能
- [x] MCSL 2 为 [最新稳定版](https://github.com/LxHTT/MCSL2/releases/latest) 或 [最新构建版](https://github.com/LxHTT/MCSL2/actions)
___

### 描述
没有确切的开发文档，很难让开发者开发新功能（

**Comment by StarryCamile** [comment]
[mcslplugininfo.txt](https://github.com/LxHTT/MCSL2/files/11382595/mcslplugininfo.txt)
这是目前的插件描述默认格式（ 与材质包差不多
我的想法是新建一个plugins文件夹（mcsl的）然后插件里有文件夹 将文件夹解压到插件文件夹内 里面有描述文件，插件的代码就用python好了，直接调用mcsl的代码（
这样的话路径就会变成 /MCSL2/plugins/插件名/描述文件、代码

**Comment by lxhtt** [comment]
和我想得不谋而合（
插一句，插件系统还是我搞吧，你的我会采纳一部分，毕竟我代码已经在纸上完整的写一遍了

**Comment by StarryCamile** [comment]
我已经在写了（
我打算就写到读取插件就停，剩下的你和我会有差别，所以你来写（
